osu.Game/Online/Rooms/MultiplayerPlaylistItem.cs

```diff
@@ -82,5 +82,24 @@ public MultiplayerPlaylistItem(PlaylistItem item)
             StarRating = item.Beatmap.StarRating;
             Freestyle = item.Freestyle;
         }
+
+        public bool Equals(MultiplayerPlaylistItem? other)
```

Comment:
Where is this equality member used? Rider is only showing me `MultiplayerSpectateButton`; is that it? If it is, can this be removed and a local `ID` equality check be used in that usage site or something?

I'm skittish about this equality check existing because it includes stuff like `StarRating` and `Expired` which is highly incidental.